android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -300,6 +301,8 @@ public boolean onMenuOrKeyboardAction(int id, boolean fromMenu) {
             ApplicationLifetime.terminate(false);
         } else if (id == R.id.set_default_browser) {
             BraveSetDefaultBrowserUtils.showBraveSetDefaultBrowserDialog(BraveActivity.this, true);
+        } else if (id == R.id.brave_talk_id) {
+            openNewOrSelectExistingTab(BRAVE_TALK_URL);
```

Comment:
It seems this method only tries to find exactly same url among exsiting tabs.
If user has a talk tab that already started call, its url is changed like https://talk.brave.com/xxxxxxxx.
Then, new talk tab will be created instead of switching to existing talk tab.
Is this what we want?

Reply:
@simonhong based on the conversion with @mattmcalister it is expected to start new tab everytime user clicks on the menu item

Reply:
@deeppandya hmm, then maybe we should not use `openNewOrSelectExistingTab()` because if there is a tab that loaded https://talk.brave.com already, it's selected instead of creating new one when user clicks the menu?